File: stacks_and_queues/stack_example.java
public class Example
{
    public static void main(String[] args)
    {
        Stack<Integer> stack = new Stack<>();

        stack.push(1);
        stack.push(2);
        stack.push(3);

        System.out.println(stack.pop());
        System.out.println(stack.pop());
        System.out.println(stack.pop());

        stack.push(5);

        if (stack.empty())
        {
            System.out.println("Stack is empty");
        }
        else
        {
            System.out.println(String.format("Stack is not empty, top is: %d", stack.peek()));
        }
    }
}

File: stacks_and_queues/stack_example.js
(function main() {
    let stack = [];

    stack.push(1);
    stack.push(2);
    stack.push(3);

    console.log(stack.pop());
    console.log(stack.pop());
    console.log(stack.pop());

    stack.push(5);

    if (!stack.length)
    {
        console.log("Stack is empty!");
    }
    else
    {
        console.log(`Stack is not empty, top is: ${stack[stack.length - 1]}`);
    }
}());